src/sass/main.scss

```diff
@@ -206,3 +206,38 @@ body {
 	&--android {
 	}
 }
+
+
+// footer
+
+.foot-container{
+	margin-top: 4rem;
+
+	& .logo{
+		padding-bottom: $fs-12;
+	}
+
+	& .center-text{
+		width: 295px;
+		margin-top: 1rem;
+		p{
+			line-height: $lh-14;
+		}
+		span{
+			font-weight: bold;
```

Comment:
use the scss variables $fw-bold and $fw-regular

Reply:
Corrected